### PR TITLE
Docs: Add guidelines for inclusive language

### PR DESCRIPTION
**Pull request by marcusolsson**
**What this PR does / why we need it**:

Adds guidelines for how to avoid using charged language.

#18841

### CODE REVIEW

**Review by daniellee — APPROVED** [review]
LGTM

The tricky part is 3rd party libraries or technologies.

**Comment by marcusolsson** [comment]
Agreed. I tried explaining it under Exceptions, using the Git master branch as example, but now realize that this should apply to all guidelines under Inclusive language.

File: contribute/style-guides/documentation-style-guide.md [hunk 1]
```diff
@@ -168,3 +168,29 @@ Two words if used as a verb, one word if used as a noun.
 
 * Set up the workspace.
 * Initial setup might take five minutes.
+
+### Inclusive language
```
Comment (oddlittlebird) [inline contribute/style-guides/documentation-style-guide.md:172]: I think this section should be an h2 and above word usage. 

Also, periods should be inside the quotation marks in this section, not outside.

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Whoops, forgot to make the earlier comment a review.

**Comment by marefr** [comment]
@oddlittlebird can you fix the changes you've suggested since Marcus is on vacation?

**Comment by oddlittlebird** [comment]
@marefr , will do

**Review by oddlittlebird — APPROVED** [review]

**Pull request merged**